### PR TITLE
Update and add tests for error message on user federation

### PR DESCRIPTION
**Pull request by Aboullos**
Fix and add tests for error messages in user federation

### CODE REVIEW

File: cypress/e2e/user_fed_ldap_test.spec.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ describe("User Federation LDAP tests", () => {
     cy.intercept("GET", "/admin/realms/master").as("getProvider");
   });
 
-  it("Should create LDAP provider from empty state", () => {
+  it.only("Should create LDAP provider from empty state", () => {
```
Comment (jonkoops) [inline cypress/e2e/user_fed_ldap_test.spec.ts:104]:
```suggestion
  it("Should create LDAP provider from empty state", () => {
```

File: cypress/e2e/user_fed_ldap_test.spec.ts [hunk 2]
```diff
@@ -362,7 +362,7 @@ describe("User Federation LDAP tests", () => {
     sidebarPage.goToUserFederation();
   });
 
-  it("Should update LDAP searching and updating settings and save", () => {
+  it.only("Should update LDAP searching and updating settings and save", () => {
```
Comment (jonkoops) [inline cypress/e2e/user_fed_ldap_test.spec.ts:365]:
```suggestion
  it("Should update LDAP searching and updating settings and save", () => {
```

File: cypress/e2e/user_fed_ldap_test.spec.ts [hunk 3]
```diff
@@ -550,7 +563,7 @@ describe("User Federation LDAP tests", () => {
     masthead.checkNotificationMessage(deletedSuccessMessage);
   });
 
-  it("Should delete LDAP provider using Settings view Action menu", () => {
+  it.only("Should delete LDAP provider using Settings view Action menu", () => {
```
Comment (jonkoops) [inline cypress/e2e/user_fed_ldap_test.spec.ts:566]:
```suggestion
  it("Should delete LDAP provider using Settings view Action menu", () => {
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: cypress/e2e/user_fed_ldap_test.spec.ts (hunk 1 above)
Comment (Aboullos) [reply cypress/e2e/user_fed_ldap_test.spec.ts:104]: Yes, this is just a draft to run those tests only, to check that there was a bug, now I already removed the only function

**Comment by ssilvert** [comment]
@Aboullos My fix is now merged.  Please rebase and see if it fixes the failing test.

**Comment by Aboullos** [comment]
> @Aboullos My fix is now merged. Please rebase and see if it fixes the failing test.

I was checking it locally, but I am getting the same error

**Comment by Aboullos** [comment]
Looks like it is working @ssilvert

**Review by jonkoops — APPROVED** [review]

File: public/resources/en/user-federation.json [hunk 4]
```diff
@@ -73,6 +73,7 @@
   "saveError": "User federation provider could not be saved: {{error}}",
   "createSuccess": "User federation provider successfully created",
   "createError": "User federation provider could not be created: {{error}}",
+  "ldapErrorValidatePasswordPolicyAvailableForWritableOnly": "Validate Password Policy is applicable only with WRITABLE edit mode",
```
Comment (edewit) [inline public/resources/en/user-federation.json:76]: this is no longer needed right? this translation is done by keycloak

File: src/components/alert/Alerts.tsx [hunk 5]
```diff
@@ -49,7 +49,7 @@ export const AlertProvider: FunctionComponent = ({ children }) => {
   const addError = (message: string, error: Error | AxiosError | string) => {
     addAlert(
       t(message, {
-        error: getErrorMessage(error),
+        error: t(getErrorMessage(error)),
```
Comment (edewit) [inline src/components/alert/Alerts.tsx:52]: also this is not needed, it can't work like this as it's missing the namespace

**Review by edewit — CHANGES_REQUESTED** [review]

**Review by edewit — APPROVED** [review]

**Pull request merged**